index.html

```diff
@@ -1,33 +1,30 @@
-<!DOCTYPE HTML>
-
-<html>
-	<head>
-		<title>Minimo Blog Challenge</title>
-		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<link rel="stylesheet" href="css/normalize.css" />
-		<link href="https://fonts.googleapis.com/css?family=Playfair+Display|Ubuntu" rel="stylesheet">
-		<link rel="stylesheet" href="css/styles.css" />
-		<script src="js/picturefill.min.js" async></script>
-	</head>
-	<body>
-		<header>
-				<img class="minimo" src="images/Minimo.jpg" alt="Minimo logo">
-			<div class="main-nav">
-				<ul>
-					<li><a href="#lifestyle">lifestyle</a></li>
-					<li><a href="#photodiary">photodiary</a></li>
-					<li><a href="#music">music</a></li>
-					<li><a href="#travel">travel</a></li>
-				</ul>
-			</div>
-			<div>
-				<picture>
-					<source srcset="images/Header_photo.jpg 1059w, images/Header_photo_rectangle.jpg 778w, images/Header_photo_square.jpg 429w" sizes="90vw" />
-					<img src="images/Header_photo.jpg" alt="picture of two women surrounded by sand" />
-				</picture>
-			</div>
-		</header>
-	</body>
-</html>
-
+<!DOCTYPE HTML>
+
+<html>
+	<head>
+		<title>Minimo Blog Challenge</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="css/normalize.css" />
+		<link href="https://fonts.googleapis.com/css?family=Playfair+Display|Ubuntu" rel="stylesheet">

```

Comment:
Bądź konekwentny. Linijkę wyżej zamknąłeś `self-closing tag` slashem na końcu, a tą już nie. To samo jest z `<img class="minimio" src="..." alt="...">` kilka linijek niżej.